modules/os2forms_fbs_handler/src/Client/FBS.php

```diff
@@ -0,0 +1,234 @@
+<?php
+
+namespace Drupal\os2forms_fbs_handler\Client;
+
+use Drupal\os2forms_fbs_handler\Client\Model\Guardian;
+use Drupal\os2forms_fbs_handler\Client\Model\Patron;
+use Fig\Http\Message\RequestMethodInterface;
+use GuzzleHttp\Client;
+
+/**
+ * Minimalistic client to create user with guardians at FBS.
+ */
+final class FBS {
```

Comment:
here and rest of the classes **final** keyword is used. Hope there is a good reason for that, otherwise i don't see a need for making classes final

Reply:
Fixed